### DISCUSSION TITLE
mengubah tipe data ke dalam bentuk date

### DISCUSSION THREAD

**Discussion by sabrinaa311** [teamalgoritma/community#327] [Q&A] [answered]
Permisi kak, izin bertanya saya sudah mencoba mengubah tipe data ke tipe date dengan lubridate, tapi saat syntax nya diterapkan untuk satu kolom maka hasilnya akan menja NA. Ini kira kira kira saya harus menggunakan syntax seperti apa ya kak untuk mengubah tipe data transaction date ke tipe data date
![Capstone project machine learning - RStudio 22_09_2022 14_53_39](https://user-images.githubusercontent.com/107026622/191690566-ae39c373-0048-40ad-a049-80e9f47f801c.png)

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @sabrinaa311 

Kolom `transaction_date` tersebut memiliki informasi dari tahun hingga detik. Maka dari itu gunakan fungsi `ymd_hms()` dari lubridate untuk convert seluruh informasi tersebut ke datetime alih-alih hanya `ymd()`. Lebih jelasnya Anda dapat mengacu pada [cheatsheet lubridate](https://evoldyn.gitlab.io/evomics-2018/ref-sheets/R_lubridate.pdf).

Jika jawaban ini dirasa membantu silahkan tandai jawaban ini sebagai "Mark as Answer", jika masih terdapat kendala silahkan reply jawaban ini agar dibantu lebih lanjut.

Sekian, Terima kasih
